liquidprompt

```diff
@@ -3311,61 +3363,67 @@ _lp_default_theme_prompt_data() {
     fi
 
     if _lp_python_env_color; then
-        LP_VENV=" $lp_python_env_color"
+        LP_VENV="$lp_python_env_color"
     else
         LP_VENV=
     fi
 
     if _lp_node_env_color; then
-        LP_NODE_VENV=" $lp_node_env_color"
+        LP_NODE_VENV="$lp_node_env_color"
     else
         LP_NODE_VENV=
     fi
 
     if _lp_ruby_env_color; then
-        LP_RUBY_VENV=" $lp_ruby_env_color"
+        LP_RUBY_VENV="$lp_ruby_env_color"
     else
         LP_RUBY_VENV=
     fi
 
     if _lp_kubernetes_context_color; then
-        LP_KUBECONTEXT=" $lp_kubernetes_context_color"
+        LP_KUBECONTEXT="$lp_kubernetes_context_color"
     else
         LP_KUBECONTEXT=
     fi
 
     if _lp_terraform_env_color; then
-        LP_TFSPACE=" $lp_terraform_env_color"
+        LP_TFSPACE="$lp_terraform_env_color"
     else
         LP_TFSPACE=
     fi
 
     if _lp_container_color; then
-        LP_CONTAINER=" $lp_container_color"
+        LP_CONTAINER="$lp_container_color"
     else
         LP_CONTAINER=
     fi
 
     if _lp_software_collections_color; then
-        LP_SCLS=" $lp_software_collections_color"
+        LP_SCLS="$lp_software_collections_color"
     else
         LP_SCLS=
     fi
 
     if _lp_aws_profile_color; then
-        LP_AWS_PROFILE=" $lp_aws_profile_color"
+        LP_AWS_PROFILE="$lp_aws_profile_color"
     else
         LP_AWS_PROFILE=
     fi
 
+    if _lp_cmake; then
+        LP_CMAKE="${lp_cmake_compiler}${LP_MARK_CMAKE}${lp_cmake_generator}${LP_MARK_CMAKE}${lp_cmake_buildtype}"
```

Comment:
This string building should not be done here, but in a `_lp_cmake_color()` function. Both to follow previous standards, and to keep reusable theming accessible to other themes.